redis_consumer/processing.py

```diff
@@ -42,10 +42,8 @@
 from deepcell_toolbox.deep_watershed import deep_watershed
 
 # import mibi pre- and post-processing functions
-from deepcell_toolbox.deep_watershed import deep_watershed_mibi
-from deepcell_toolbox.deep_watershed import format_output_multiplex
-from deepcell_toolbox.deep_watershed import deep_watershed_subcellular
-from deepcell_toolbox.processing import phase_preprocess
+from deepcell_toolbox.multiplex_utils import \
+    multiplex_preprocess, multiplex_postprocess, format_output_multiplex
```

Comment:
I don't like to use the `\` if its avoidable.

```suggestion
from deepcell_toolbox.multiplex_utils import format_output_multiplex
from deepcell_toolbox.multiplex_utils import multiplex_preprocess
from deepcell_toolbox.multiplex_utils import multiplex_postprocess
```